packages/react/src/components/Swap/Royalties.tsx

```diff
@@ -19,14 +19,21 @@ export function Royalties({
   isBuying: boolean;
   formRef: React.MutableRefObject<HTMLInputElement>;
 }) {
+  const { metadata: baseMeta, loading: baseMetaLoading } = useTokenMetadata(
+    tokenBonding.baseMint
+  );
+
+  const { metadata: targetMeta, loading: targetMetaLoading } = useTokenMetadata(
```

Comment:
Are baseMetaLoading and targetMetaLoading used anywhere?

Reply:
Looks like no